azure-communication-ui/azure-communication-ui/src/main/java/com/azure/android/communication/ui/presentation/fragment/setup/components/SetupControlBarViewModel.kt

```diff
@@ -26,6 +26,9 @@ internal class SetupControlBarViewModel(
     private lateinit var audioDeviceSelectionStatusStateFlow: MutableStateFlow<AudioDeviceSelectionStatus>
     private lateinit var callingStatusStateFlow: MutableStateFlow<CallingStatus>
 
+    // Does not need StateFlow as tied to AudioDeviceSelectionStatus updates
```

Comment:
I think we can remove this comment

Reply:
Deleted this member field (and comment), as it all comes through AudioState and BluetoothState now.